tests/UI/pages/BO/design/positions/index.js

```diff
@@ -0,0 +1,32 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+class Positions extends BOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Positions';
+
+    // Selectors
+    this.searchInput = '#hook-search';
+    this.modulePositionForm = '#module-positions-form';
+    this.searchResultHookNameSpan = `${this.modulePositionForm} section[style] header span span`;
```

Comment:
That's such a beautiful selector... can we do something about it ? :laughing:

Reply:
Yes, we can add a specific class for each hook, we will fix it in develop branch :)